lookout/style/format/benchmarks/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+The module contains the code and data for the format style-analyzer benchmarking and benchmarks
+generation. To use it git should be installed in your system.
+"""
+# flake8: noqa: F401
+from lookout.style.format.benchmarks.generate_styles import generate_styles_entry
```

Comment:
We keep one to one mapping between modules and file structure everywhere so we should remove this import.